File: src/main/java/org/kite9/diagram/model/style/DiagramElementSizing.java
package org.kite9.diagram.model.style;

/**
 * This is used to choose the right approach for laying out the diagram element.
 * Currently, this is only used for containers.
 * 
 * When elements are part of a grid, MINIMIZE has priority over MAXIMIZE.
 * 
 * @author robmoffat
 *
 */
public enum DiagramElementSizing {

	MINIMIZE, MAXIMIZE 	// these apply to Container elements
	
}

File: visualization/kite9-visualization-common/src/visMain/kotlin/org/kite9/diagram/model/style/DiagramElementSizing.kt
package org.kite9.diagram.model.style

/**
 * This is used to choose the right approach for laying out the diagram element.
 *
 * When elements are part of a grid, MINIMIZE has priority over MAXIMIZE.
 *
 * @author robmoffat
 */
enum class DiagramElementSizing {
    MINIMIZE, MAXIMIZE
}